docs/Shipment.java

```diff
@@ -0,0 +1,53 @@
+public class Shipment(){
+	
+	String shipmentId; 
+	String shipmentMethod;	
+	String currentWarehouseId;
+	float shipmentWeight;
+	long receivedAt;
+	
+	//No argument constructor not provided
+	public Shipment(String shipId, String shipMthd, float weight){	
+		shipmentId = shipID;
+		shipmentMethod = shipMthd;
+		shipmentWeight = weight;
+	}
+	
+	public Shipment(String shipId, String shipMthd, String warehouse, float weight, long received){
+	
+		shipmentId = shipID;
+		shipmentMethod = shipMthd;
+		currentWarehouseId = 
+		shipmentWeight = weight;
+		receivedAt = received;		
+	}
+
+	//Basic getters and setters
```

Comment:
I'm not sure we need or want these boilerplate setters and getters at this point, only those that are called by the software to this point. On the one hand, they may may not hurt anything since we could assume that only trusted applications can call the methods and get the information. On the other hand, it could be a security hole and we don't have requirements for them at this point, which is scope creep.

A good compromise could be to comment out the methods we aren't using at this point, and to set them private until they need to be public.

Also, if we're going to include some, it's probably good to include all of them, and with corresponding names (getX, setX).